stablehlo/dialect/VhloTypes.td

```diff
@@ -119,6 +119,9 @@ def VHLO_FunctionV1 : VHLO_TypeDef<"FunctionV1", "func_v1", "0.9.0", "current">
 // dynamism RFC.
 def VHLO_IndexV1 : VHLO_TypeDef<"IndexV1", "index_v1", "0.9.0", "current">;
 
+// Corresponds to the 'si2' FloatType from the StableHLO spec.
```

Comment:
s/FloatType/IntegerType/
here and below

Reply:
Done. Good catch!